patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -285,3 +285,38 @@
          }
  
          this.func_184602_cy();
+@@ -2626,4 +2707,34 @@
+     {
+         return true;
+     }
++
++    // FORGE
++    private final net.minecraftforge.items.IItemHandlerModifiable handHandler = new net.minecraftforge.items.ItemStackHandler(field_184630_bs);
++    private final net.minecraftforge.items.IItemHandlerModifiable armorHandler = new net.minecraftforge.items.ItemStackHandler(field_184631_bt);
++    private final net.minecraftforge.items.IItemHandler joinedHandler = new net.minecraftforge.items.wrapper.CombinedInvWrapper(armorHandler, handHandler);
++
++    @Override
++    public <T> T getCapability(net.minecraftforge.common.capabilities.Capability<T> capability, net.minecraft.util.EnumFacing facing)
++    {
++        if (capability == net.minecraftforge.items.CapabilityItemHandler.ITEM_HANDLER_CAPABILITY)
++        {
++            if (facing == net.minecraft.util.EnumFacing.UP)
++            {
++                return (T) armorHandler;
++            } else if (facing.func_176740_k().func_176722_c())
```

Comment:
No, No it's not.
In these cases using no Braces is preferred as it keeps patch sizes down.
You could also in theory have the return on the same line.